### PR TITLE
AT_05.05_006 | New Item | Create a new Multibranch Pipeline

### PR DESCRIPTION
**Pull request by Parizat-Ismailova**
https://trello.com/c/XDSJSeOs/630-at0505006-new-item-create-a-new-multibranch-pipeline

### CODE REVIEW

**Review by matvlesia — COMMENTED** [review]
Hi! You should not use a hardcode such as 'http://localhost:8080/' in your test. Because the number of host could be different for everyone

File: cypress/e2e/createMultibranchPipeline.cy.js [hunk 1]
```diff
@@ -14,4 +17,17 @@ describe('New Item Create a new Multibranch Pipeline', () => {
             .should('exist')
             .and('include.text', pipelineName.namePipeline)
     })
+
+    it('AT_05.05_006 | New Item | Create a new Multibranch Pipeline', ()=>{
+        cy.get(':nth-child(1) > .task-link-wrapper > .task-link').click()
+        cy.url().should('includes','/view/all/newJob')
+        cy.get('#name').type(multibranchPipeline.newMultiPipeline)
+        cy.contains('Multibranch Pipeline').click()
```
Comment (DariaMas7) [inline cypress/e2e/createMultibranchPipeline.cy.js:25]: Multibranch Pipeline to fixtures

File: cypress/e2e/createMultibranchPipeline.cy.js [hunk 2]
```diff
@@ -14,4 +17,17 @@ describe('New Item Create a new Multibranch Pipeline', () => {
...
+        cy.get('#ok-button').click()
+        cy.url().should('includes','/configure')
+        cy.get('button[name="Submit"]').click()
+        cy.contains(multibranchPipeline.newMultiPipeline).should('be.visible')
+        cy.contains('Dashboard').click()
```
Comment (DariaMas7) [inline cypress/e2e/createMultibranchPipeline.cy.js:30]: Dashboard to fixtures

File: cypress/e2e/createMultibranchPipeline.cy.js [hunk 3]
```diff
@@ -14,4 +17,17 @@ describe('New Item Create a new Multibranch Pipeline', () => {
...
+        cy.url().should('includes','http://localhost:8080/')
```
Comment (DariaMas7) [inline cypress/e2e/createMultibranchPipeline.cy.js:31]: localhost:8080/ we dont use it

**Review by IrynaLyakhova — APPROVED** [review]

**Review by DariaMas7 — APPROVED** [review]

File: cypress/e2e/createMultibranchPipeline.cy.js (hunk 1 above)
Comment (Parizat-Ismailova) [reply cypress/e2e/createMultibranchPipeline.cy.js]: Done, thank you

File: cypress/e2e/createMultibranchPipeline.cy.js (hunk 2 above)
Comment (Parizat-Ismailova) [reply cypress/e2e/createMultibranchPipeline.cy.js]: Done, thank you

File: cypress/e2e/createMultibranchPipeline.cy.js (hunk 3 above)
Comment (Parizat-Ismailova) [reply cypress/e2e/createMultibranchPipeline.cy.js]: Done, thank you

**Comment by Parizat-Ismailova** [comment]
> Hi! You should not use a hardcode such as 'http://localhost:8080/' in your test. Because the number of host could be different for everyone

Done, thank you

**Review by IrynaLyakhova — APPROVED** [review]

**Pull request merged**